Lesson-7/LinkedSprite.svg

```diff
@@ -0,0 +1,9 @@
+<svg xmlns="http://www.w3.org/2000/svg"><defs><style>.icon { display: none; }.icon:target { display: inline; }</style></defs><svg viewBox="0 0 75.5 91.833336"><g id="headphones" class="icon">
+<path d="M67.1667 54.3333V71H58.8333V54.3333H67.1667ZM17.1667 54.3333V71H13C10.7083 71 8.83333 69.125 8.83333 66.8333V54.3333H17.1667ZM38 0.166626C17.2917 0.166626 0.5 16.9583 0.5 37.6666V66.8333C0.5 73.75 6.08333 79.3333 13 79.3333H25.5V46H8.83333V37.6666C8.83333 21.5416 21.875 8.49996 38 8.49996C54.125 8.49996 67.1667 21.5416 67.1667 37.6666V46H50.5V79.3333H67.1667V83.5H38V91.8333H63C69.9167 91.8333 75.5 86.25 75.5 79.3333V37.6666C75.5 16.9583 58.7083 0.166626 38 0.166626Z" fill="#3077C6"/>
+</g></svg><svg viewBox="0 0 83.666664 83.666664"><g id="communicate" class="icon">
+<path d="M54.5 8.66671V37.8334H13.5417L8.66666 42.7084V8.66671H54.5ZM58.6667 0.333374H4.5C2.20833 0.333374 0.333328 2.20837 0.333328 4.50004V62.8334L17 46.1667H58.6667C60.9583 46.1667 62.8333 44.2917 62.8333 42V4.50004C62.8333 2.20837 60.9583 0.333374 58.6667 0.333374ZM79.5 17H71.1667V54.5H17V62.8334C17 65.125 18.875 67 21.1667 67H67L83.6667 83.6667V21.1667C83.6667 18.875 81.7917 17 79.5 17Z" fill="#3077C6"/>
+</g></svg><svg viewBox="0 0 100 67.333336"><g id="gadgets" class="icon">
+<path d="M16.6667 8.99996H91.6667V0.666626H16.6667C12.0833 0.666626 8.33333 4.41663 8.33333 8.99996V54.8333H0V67.3333H58.3333V54.8333H16.6667V8.99996ZM95.8333 17.3333H70.8333C68.5417 17.3333 66.6667 19.2083 66.6667 21.5V63.1666C66.6667 65.4583 68.5417 67.3333 70.8333 67.3333H95.8333C98.125 67.3333 100 65.4583 100 63.1666V21.5C100 19.2083 98.125 17.3333 95.8333 17.3333ZM91.6667 54.8333H75V25.6666H91.6667V54.8333Z" fill="#3077C6"/>
+</g></svg><svg viewBox="0 0 91.833336 83.5"><g id="like" class="icon">
+<path d="M83.5 29.3333H57.2083L61.1667 10.2916L61.2917 8.95829C61.2917 7.24996 60.5833 5.66663 59.4583 4.54163L55.0417 0.166626L27.625 27.625C26.0833 29.125 25.1667 31.2083 25.1667 33.5V75.1666C25.1667 79.75 28.9167 83.5 33.5 83.5H71C74.4583 83.5 77.4167 81.4166 78.6667 78.4166L91.25 49.0416C91.625 48.0833 91.8333 47.0833 91.8333 46V37.6666C91.8333 33.0833 88.0833 29.3333 83.5 29.3333ZM83.5 46L71 75.1666H33.5V33.5L51.5833 15.4166L46.9583 37.6666H83.5V46ZM0.166656 33.5H16.8333V83.5H0.166656V33.5Z" fill="#3077C6"/>
+</g></svg></svg>
```

Comment:
You have LinkedSprite.svg file, Sprite.svg file, 1,2,3,4.svg files. It should only on one.